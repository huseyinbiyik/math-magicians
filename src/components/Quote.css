.quote-section {
  width: 90%;
  display: flex;
  justify-content: center;
  align-items: center;
  margin: 180px auto;
}

.quote-section > p {
  font-size: 48px;
}
