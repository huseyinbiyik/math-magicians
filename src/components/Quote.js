import React from "react";
import './Quote.css'

const Quote = () => {
  return (
    <section className="quote-section">
      <p>
        Math. The only place where people can buy 64 watermelons and no one
        wonders whyâ€¦
      </p>
    </section>
  );
};

export default Quote;
