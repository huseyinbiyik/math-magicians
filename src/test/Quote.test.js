import renderer from 'react-test-renderer';
import Quote from '../components/Quote';

describe('Test for quote page', () => {
  it('It should render quote component', () => {
    const tree = renderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
