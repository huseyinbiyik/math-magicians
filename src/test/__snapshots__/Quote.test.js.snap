// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for quote page It should render quote component 1`] = `
<section
  className="quote-section"
>
  <p>
    Math. The only place where people can buy 64 watermelons and no one wonders whyâ€¦
  </p>
</section>
`;
